docs/admin/access.rst

```diff
@@ -429,8 +416,8 @@ Default groups and roles
 After installation, a default set of groups is created (see :ref:`default-groups`).
 
 These roles and groups are created upon installation. The built-in roles are
-always kept up to date by the database migration when upgrading. You can’t
-actually change them, please define a new role if you want to define your own
+always kept up to date by the database migration when upgrading. Custom changes
+are not lost. Please define a new role if you want to define your own
```

Comment:
Editing of built-in roles is not possible in the UI and any changes are lost on upgrade.

Reply:
I'll just put that in verbatim.